### ISSUE TITLE
Error while installing 

### ISSUE THREAD

**Issue by Oelsakha** [antismash/antismash#490]
I tried  to install manually with `python setup.py install ` and I got this error :TypeError: expected string or bytes-like object 
I tried  t o use pip `pip install ./antismash-6.1.1` gave me this error Processing ./antismash-6.1.1
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [30 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/Users/mo/antismash-6.1.1/setup.py", line 98, in <module>
          setup(
        File "/Users/mo/miniconda3/lib/python3.9/site-packages/setuptools/__init__.py", line 86, in setup
          _install_setup_requires(attrs)
        File "/Users/mo/miniconda3/lib/python3.9/site-packages/setuptools/__init__.py", line 75, in _install_setup_requires
          dist = MinimalDistribution(attrs)
        File "/Users/mo/miniconda3/lib/python3.9/site-packages/setuptools/__init__.py", line 57, in __init__
          super().__init__(filtered)
        File "/Users/mo/miniconda3/lib/python3.9/site-packages/setuptools/dist.py", line 474, in __init__
          for ep in metadata.entry_points(group='distutils.setup_keywords'):
        File "/Users/mo/miniconda3/lib/python3.9/site-packages/setuptools/_vendor/importlib_metadata/__init__.py", line 999, in entry_points
          return SelectableGroups.load(eps).select(**params)
        File "/Users/mo/miniconda3/lib/python3.9/site-packages/setuptools/_vendor/importlib_metadata/__init__.py", line 449, in load
          ordered = sorted(eps, key=by_group)
        File "/Users/mo/miniconda3/lib/python3.9/site-packages/setuptools/_vendor/importlib_metadata/__init__.py", line 996, in <genexpr>
          eps = itertools.chain.from_iterable(
        File "/Users/mo/miniconda3/lib/python3.9/site-packages/setuptools/_vendor/importlib_metadata/_itertools.py", line 16, in unique_everseen
          k = key(element)
        File "/Users/mo/miniconda3/lib/python3.9/site-packages/setuptools/_vendor/importlib_metadata/__init__.py", line 931, in _normalized_name
          return self._name_from_stem(stem) or super()._normalized_name
        File "/Users/mo/miniconda3/lib/python3.9/site-packages/setuptools/_vendor/importlib_metadata/__init__.py", line 600, in _normalized_name
          return Prepared.normalize(self.name)
        File "/Users/mo/miniconda3/lib/python3.9/site-packages/setuptools/_vendor/importlib_metadata/__init__.py", line 855, in normalize
          return re.sub(r"[-_.]+", "-", name).lower().replace('-', '_')
        File "/Users/mo/miniconda3/lib/python3.9/re.py", line 210, in sub
          return _compile(pattern, flags).sub(repl, string, count)
      TypeError: expected string or bytes-like object
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

I am using OS system.

**Comment by kblin** [comment]
I take it that when you say OS you mean a Mac. How did you set up your conda environment?

**Comment by Oelsakha** [comment]
I used 
`conda create -n antismash antismash`
And I got this error  so I tried manually. 
Collecting package metadata (current_repodata.json): done
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - antismash

Current channels:

  - https://conda.anaconda.org/conda-forge/osx-64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://repo.anaconda.com/pkgs/main/osx-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/osx-64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.

**Comment by kblin** [comment]
You didn't install bioconda, which is where the antiSMASH package and lots of its dependencies live. Please see https://bioconda.github.io/index.html for instructions on how to do that.

**Comment by Oelsakha** [comment]
I installed bioconda and tried again. 
I got this error:

Collecting package metadata (current_repodata.json): done
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: | 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                                                                                   

UnsatisfiableError: The following specifications were found to be incompatible with your system:

  - feature:/osx-64::__osx==12.6=0
  - antismash -> matplotlib-base -> __osx[version='>=10.12']

Your installed version is: 12.6

Note that strict channel priority may have removed packages required for satisfiability.
